### PR TITLE
Payments+unreserve+connect

### PR DESCRIPTION
**Pull request by arsidada**

### CODE REVIEW

File: src/Components/Dashboard/Cards/CoffeeChatSelfCard.js [hunk 1]
```diff
@@ -673,15 +723,32 @@ class CoffeeChatSelfCard extends Component {
...
+                    ))}
+                  </span>
+                  <Grid
+                    container
+                    item
+                    xs={6}
+                    spacing={0}
+                    alignItems="flex-start"
+                    justify="flex-start"
+                  ></Grid>
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/Cards/CoffeeChatSelfCard.js:751]: Empty grid component?

File: src/Components/Dashboard/UserProfile.js [hunk 2]
```diff
@@ -661,28 +693,46 @@ class Landing extends Component {
...
-          {jwtDecode(localStorage.getItem("idToken"))["custom:user_type"] !==
-          "FREE" ? (
-            <Button
-              className={classes.button1}
-              variant="contained"
-              onClick={this.postJob}
-            >
-              Post a Job
-            </Button>
-          ) : null}
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/UserProfile.js:685]: Do we not want the Post a Job button behind this user type filter?

**Review by ahmedhamodi — APPROVED** [review]
Overall this looks amazing. A couple of small comments.

Would love to hop on a call to walk through a demo of the Stripe work. I can make some time tonight so let me know!

File: src/Components/Dashboard/Cards/CoffeeChatSelfCard.js (hunk 1 above)
Comment (arsidada) [reply src/Components/Dashboard/Cards/CoffeeChatSelfCard.js]: oops. fixed

File: src/Components/Dashboard/UserProfile.js (hunk 2 above)
Comment (arsidada) [reply src/Components/Dashboard/UserProfile.js]: yeah. I misunderstood this. fixed!

**Pull request merged**